### PR TITLE
fix: restore/update cloud-functions-python-http

### PR DESCRIPTION
**Pull request by PicardParis**
- [PR#9497](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/9497) broke the codelab [cloud-functions-python-http](https://codelabs.developers.google.com/codelabs/cloud-functions-python-http)
- The codelab + source code have been updated with explicit typing (`flask.Request`/`flask.Response` inputs/outputs)

### CODE REVIEW

**Review by kweinmeister — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -90,6 +90,7 @@
 # Self-service
 /appengine/**/*                           @GoogleCloudPlatform/serverless-runtimes  @GoogleCloudPlatform/python-samples-reviewers
 /appengine/standard_python3/spanner/*     @GoogleCloudPlatform/api-spanner-python @GoogleCloudPlatform/python-samples-reviewers
+/codelabs/**/*                            @GoogleCloudPlatform/codelabs-contributors @GoogleCloudPlatform/python-samples-reviewers
```
Comment (rsamborski) [inline .github/CODEOWNERS:93]: @dandhlee Can you please add this group with write access to the repo?

**Review by engelke — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (dandhlee) [reply .github/CODEOWNERS:93]: Done! Sorry for the delay.

Comment (StacheClubTest) [reply .github/CODEOWNERS]: Thanks Dan!

Comment (PicardParis) [reply .github/CODEOWNERS]: Thanks Dan!

**Pull request merged**